build_docs.sh

```diff
@@ -1,11 +1,24 @@
 #!/bin/bash
+# To build manually run "sudo sh build_docs.sh"
+# As MSAL is a mixed Objective-C and Swift project we need to use SourceKitten to
+# first extract the doc comments from the source before building with Jazzy. For more
+# details see https://github.com/realm/jazzy?tab=readme-ov-file#mixed-objective-c--swift
 gem install jazzy
 echo -e "Copying MSAL public files"
 mkdir docs.temp
 mkdir docs.temp/MSAL
 cp `find MSAL/src/public` docs.temp/MSAL
+cp `find MSAL/src/native_auth/public` docs.temp/MSAL
 cp README.md docs.temp/
-cd docs.temp
+
 echo -e "Generating MSAL documentation"
-jazzy --objc --umbrella-header MSAL/MSAL.h --framework-root . --sdk iphonesimulator --author Microsoft\ Corporation --author_url https://aka.ms/azuread  --github_url https://github.com/AzureAD/microsoft-authentication-library-for-objc --theme fullwidth
+# Generate Swift SourceKitten output
+sourcekitten doc -- -workspace MSAL.xcworkspace -scheme "MSAL (iOS Framework)" -configuration Debug RUN_CLANG_STATIC_ANALYZER=NO -sdk iphonesimulator CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO -destination 'platform=iOS Simulator' > docs.temp/swiftDoc.json
+
+# Generate Objective-C SourceKitten output
+cd docs.temp
+sourcekitten doc --objc $(pwd)/MSAL/MSAL.h -- -x objective-c  -isysroot $(xcrun --show-sdk-path --sdk iphonesimulator) \-I $(pwd) -fmodules > objcDoc.json
+cd ..
```

Comment:
compared to the previous version of the script, we're moving to the parent directory. Is it expected?

Reply:
The script is the same as in previous PR and indeed different than the original one in this regard if that's what you meant - https://github.com/AzureAD/microsoft-authentication-library-for-objc/blob/9924950263e54dad10a426f42911f9044af27bfb/build_docs.sh and I've re-tested it. We have to run sourcekitten inside docs.tem but jazzy has to be run in the root directory as the xcodebuild (that jazzy does behind the scenes) has to be done from the root directory.